frontends/web/src/locales/en/app.json

```diff
@@ -37,6 +37,7 @@
     "balance": "Balance",
     "exportSummary": "Export accounts summary to downloads folder as CSV file",
     "fiatBalance": "Fiat balance",
+    "loadingAccounts": "Loading accounts",
     "name": "Account name",
     "noAccount": "There are no accounts to show.",
```

Comment:
Not sure if we still need "noAccount" or when to show it

Reply:
my grep shows `noAccount` is also used in [frontends/web/src/routes/buy/info.tsx](https://github.com/digitalbitbox/bitbox-wallet-app/blob/c6aab96/frontends/web/src/routes/buy/info.tsx#L207) although in the buy context maybe a better text would be: "There are no suitable accounts to choose from".

Reply:
update, but I am still not sure if there ever is a state on the accountSummary where we should show the noAccount (There are no accounts to show.) message.